public/actions/wrapKeyInCertificate.ts

```diff
@@ -0,0 +1,14 @@
+import { NIST_P256_CERTIFICATE, NIST_P256_PRIVATE_KEY } from "../utils/certificates";
+
+export const wrapKeyInCertificate = (): {
+  certBytes: Uint8Array;
+  privateKeyBytes: Uint8Array;
+} => {
+  const encoder = new TextEncoder();
+  const certBytes = encoder.encode(NIST_P256_CERTIFICATE);
+  const privateKeyBytes = encoder.encode(NIST_P256_PRIVATE_KEY);
+  return {
+    certBytes,
+    privateKeyBytes,
+  };
+};
```

Comment:
this code does not wrapp key in cert - it just load cert and private key as Uint8Array in order to impot them in KMS. I don't think that this should be a code block - maybe just in the React part